src/test/java/org/jacp/controller/QuestionControllerTest.java

```diff
@@ -0,0 +1,82 @@
+package org.jacp.controller;
+
+import org.jacp.dto.QuestionDto;
+import org.jacp.entity.QuestionEntity;
+import org.jacp.mapper.QuestionMapper;
+import org.jacp.service.QuestionService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+/**
+ * @author saffchen created on 10.08.2023
+ */
+@ExtendWith(MockitoExtension.class)
+class QuestionControllerTest {
+
+    static final String URL = "/api/v1/questions";
+
+    @Mock
+    private QuestionMapper questionMapper;
```

Comment:
Let's inject real mapper, not this mock. Otherwise you cannot handle incorrect mapping to response.

Reply:
Okay, I've found you separated the tests for mapper and controller, but probably it's not required, you can check everything in controller. I don't insist on that, on your choise.